### ISSUE TITLE
Plot ALL dependent parameters simultaneously from single dataset

### ISSUE THREAD

**Issue by astafan8** [toolsforexperiments/plottr#19]
Original request:
> Plottr does not show plots for multiple measured parts of data
> 
> Do a measurement of several dependent parameters, only the first one will show up in plottr 

Added comment:
> correct, but "first" is a default selection. User can select which dependent parameter to plot during the live plotting. So perhaps this is not a bug, but a feature request to plot dependent parameters at the same time on different plots :)

Feel free to discuss/close/implement/etc...

**Comment by wpfff** [comment]
Good point -- in the new version, you can plot all *compatible* dependents at the same time, i.e, data that have the same axes. 
It's probably worth looking into an option that plots really *all* data.
